package.json

```diff
@@ -4,11 +4,12 @@
   "description": "",
   "main": "app.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest",
```

Comment:
I can see some errors during test execution:
`UnhandledPromiseRejectionWarning: SequelizeHostNotFoundError: getaddrinfo ENOTFOUND undefined
(Use `node --trace-warnings ...` to show where the warning was created)`

It means that Sequelize tries to connect to DB. This dependencies should be mocked in unit tests, as we don't want to test integrations

Reply:
Done